descriptors/ModuleDescriptor-template.json

```diff
@@ -116,7 +291,7 @@
           "permissionsRequired": ["inventory.items.item.put"],
           "modulePermissions": [
             "inventory-storage.items.item.get",
-            "inventory-storage.items.collection.get",
+            "inventory-storage.items.collections.get",
```

Comment:
Please revert this.

Reply:
Add 119 and remove 294, right?

Reply:
Replace `collections` with `collection`.
```suggestion
            "inventory-storage.items.collection.get",
```